auth-web/src/components/auth/manage-business/StartNewBusinessHelp.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <section>
```

Comment:
I think it's good practice for components to have an id on their top element (or a classname if there can be multiple copies of the component).

PS I like the id to match the component name.